cmd/hypper/install.go

```diff
@@ -78,6 +100,20 @@ func runInstall(args []string, client *action.Install, valueOpts *values.Options
 		return nil, err
 	}
 
+	if settings.NamespaceFromFlag {
+		client.Namespace = settings.Namespace()
+	} else {
+		client.SetNamespace(chartRequested, settings.Namespace())
+	}
```

Comment:
Could we have a helper package for this? I think this is needed in several places.

Reply:
:thinking:  I thought about that, providing this logic in a library function. Thing is, that the Helm library methods and functions expect the user to find out which `ReleaseName` and `Namespace` to use on their own, and once they know them, call the library functions with them. So I followed the same pattern.

This is because it is Hypper's cli the one that has the priorities of "first --namespace flag, then namespace annotations, then default namespace in kube context". Someone else using the Hypper libraries could decide on different priorities. Hence why in my opinion this shouldn't be provided on hypper libs.

We could provide it on a cmd/hypper function, but since I have yet to find a second usage for it, I felt it as is.

Reply:
Sounds reasonable.